client/src/Components/Layout/admin/Home/Home.jsx

```diff
@@ -8,6 +8,7 @@ import LoaderProgress from '../../../Common/LoaderProgress';
 import EventDefaultImg from '../../../../assets/eventDefaultimg.svg';
 
 import HomeStyle from './HomeStyle';
+import Axios from 'axios';
```

Comment:
lower case `axios` as it's a library, not a react component or class

Reply:
done